src/components/Books.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = ({ title, author }) => (
+  <ul>
+    <div>{title}</div>
+    <div>{author}</div>
+    <button type="button">Remove</button>
+  </ul>
```

Comment:
- [ ] Please rename this component to a singular `Book`. This is the convention because this component represents a single book which has a title and an author 👍

Reply:
done

Reply:
This change has been done well.